lib/common/client/middleware/request/remove_cookies.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module Common
+  module Client
+    module Middleware
+      module Request
+        # must be added right before http_client adapter
+        class RemoveCookies < Faraday::Middleware
+          def call(env)
+            @app.client.cookie_manager.cookies = []
```

Comment:
https://github.com/nahi/httpclient/blob/master/lib/httpclient.rb#L183 recommends just setting the cookie manager to `nil` to disable this functionality